test/test_binary_ufuncs.py

```diff
@@ -1026,19 +1030,22 @@ def test_maximum_minimum_float(self, device, dtype):
             a_tensor = torch.from_numpy(a_np).to(device=device, dtype=dtype)
             b_tensor = torch.from_numpy(b_np).to(device=device, dtype=dtype)
             tensor_result = torch_op(a_tensor, b_tensor)
-            alias_result = alias(a_tensor, b_tensor)
             out = torch.empty_like(a_tensor)
             torch_op(a_tensor, b_tensor, out=out)
 
-            self.assertEqual(alias_result, tensor_result)
+            if alias is not None:
+                alias_result = alias(a_tensor, b_tensor)
+                self.assertEqual(alias_result, tensor_result)
+
             self.assertEqual(tensor_result, numpy_result)
             self.assertEqual(out, numpy_result)
 
     @dtypes(*(torch.testing.get_all_fp_dtypes()))
     def test_maximum_minimum_float_nan_and_inf(self, device, dtype):
         # np.maximum and np.minimum functions compare input arrays element-wisely.
         # if one of the elements being compared is a NaN, then that element is returned.
-        ops = ((torch.maximum, torch.max, np.maximum), (torch.minimum, torch.min, np.minimum))
+        ops = ((torch.maximum, torch.max, np.maximum), (torch.minimum, torch.min, np.minimum),
+               (torch.fmax, None, np.fmax), (torch.fmin, None, np.fmin))
         a_vals = (float('inf'), -float('inf'), float('nan'), float('nan'))
         b_vals = (-float('inf'), float('inf'), float('inf'), float('nan'))
```

Comment:
We should also test a combination of nan and non-nan values to ensure we cover the cases where the value returned should come from one tensor vs the other.

Reply:
I think the current test does check this with `a_vals` and `b_vals`?

Reply:
I think the only comparison is between float('nan') and float('inf') but what about where the only nan value is in the second tensor?

Reply:
Good point. Looks like the test could be more thorough.

Reply:
I've add the case of `float('inf')` and `float('nan')` to prove the situation of only nan value in the second tensor.